### PR TITLE
Fix WorkingSet calculation

### LINKED ISSUE #685: Node memory is returning 9223372036854775807
9223372036854775807 is the largest int64

### PR DESCRIPTION
**Pull request by vmarmol**
Fixes #685

### CODE REVIEW

File: container/libcontainer/helpers.go [hunk 1]
```diff
@@ -186,10 +186,21 @@ func toContainerStats2(s *cgroups.Stats, ret *info.ContainerStats) {
...
+			workingSet = 0
+		} else {
+			workingSet -= v
 		}
+
+		if v, ok := s.MemoryStats.Stats["total_inactive_file"]; ok {
+			if workingSet < v {
+				workingSet = 0
+			} else {
+				workingSet -= v
```
Comment (rjnagal) [inline container/libcontainer/helpers.go]: you need to check workingSet to be non-negative after this deduction.

Comment (vmarmol) [inline container/libcontainer/helpers.go]: It's a `uint64` so unfortunately I can't check. I check whether the deduction would make it negative beforehand and just set it to zero. That should work I think?

Comment (rjnagal) [inline container/libcontainer/helpers.go]: ah, this should be okay then.

LGTM, will merge on green

**Pull request merged**